student-test/assignment1/Test_validate_username.c

```diff
@@ -14,9 +14,15 @@
 */
 void test_validate_my_username()
 {
-    /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
-     * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+
+    // Verify your /conf/username.txt config file and my_username() functions are setup properly
+
+    // 1)
+    const char * hardcoded_username = my_username();
+    
+    // 2)
+    const char * received_username = malloc_username_from_conf_file();
+    
+    // 3)
+    TEST_ASSERT_TRUE_MESSAGE(hardcoded_username, received_username);
```

Comment:
@erpe9416 what happens to the string returned by `malloc_username_from_conf_file` here?